hash/msx1_cass.xml

```diff
@@ -13979,31 +14725,32 @@ license:CC0-1.0
 		</part>
 	</software>
 
-	<software name="swiv"> <!-- needs 128k -->
+	<software name="swiv"> <!-- needs 128k, should be moved to MSX2 softlist? -->
 		<description>SWIV (Spain)</description>
 		<year>1991</year>
 		<publisher>Dro Soft</publisher>
-		<info name="usage" value="Load with RUN&quot;CAS:&quot;"/>
+		<info name="serial" value="4 SV 288"/>
+		<info name="usage" value="Load with RUN&quot;CAS:&quot;. Requires a system with 128KB."/>
 
-		<part name="cass1" interface="msx_cass">
+		<part name="cass1a" interface="msx_cass">
 			<feature name="part_id" value="Side A"/>
 			<dataarea name="cass" size="95641">
 				<rom name="swiv (1991)(dro soft)(es)(side a)[needs 128k][run'cas-'].cas" size="95641" crc="5399fd47" sha1="5c6759b545d619c5fd600d1948504da40297b059"/>
 			</dataarea>
 		</part>
-		<part name="cass2" interface="msx_cass">
+		<part name="cass1b" interface="msx_cass">
 			<feature name="part_id" value="Side B"/>
 			<dataarea name="cass" size="67450">
 				<rom name="swiv (1991)(dro soft)(es)(side b)[needs 128k][run'cas-'].cas" size="67450" crc="8db8879f" sha1="f4f97825cbf99652731199b8fcb8506538e691fc"/>
 			</dataarea>
 		</part>
 	</software>
 
-	<software name="swiva" cloneof="swiv"> <!-- needs 128k -->
+	<software name="swiva" cloneof="swiv"> <!-- needs 128k, should be moved to MSX2 softlist? -->
 		<description>SWIV (Spain, alt)</description>
 		<year>1991</year>
 		<publisher>Dro Soft</publisher>
-		<info name="usage" value="Load with RUN&quot;CAS:&quot;"/>
+		<info name="usage" value="Load with RUN&quot;CAS:&quot;. Requires a system with 128KB."/>
```

Comment:
Can you please be explicit and say 128kB RAM? Saying 128KB on its own is potentially ambiguous.

Reply:
Good point, changed these to 128KB RAM.